includes/wikia/services/MediaQueryService.class.php

```diff
@@ -398,7 +398,6 @@ public static function getRecentlyUploadedAsMediaTable( $limit ) {
 	 *
 	 * @param string $type What type of videos to return.  Valid options are:
 	 *                     - all     : Show all videos (DEFAULT)
-	 *                     - premium : Show only premium videos
```

Comment:
let's remove `type` param if it only handles 1 type

Reply:
Fixed in 4c824c7